pkg/model/replica_entity.go

```diff
@@ -57,50 +50,26 @@ type replicaEntity struct {
 var replicaNum int
 
 func (re *replicaEntity) Activate() {
-	endpoints, err := re.kubernetesClient.CoreV1().Endpoints("skenario").Get("Skenario Revision", metav1.GetOptions{})
-	if err != nil {
-		panic(err.Error())
-	}
-
-	endpoints.Subsets[0].Addresses = append(endpoints.Subsets[0].Addresses, re.endpointAddress)
-
-	updatedEndpoints, err := re.kubernetesClient.CoreV1().Endpoints("skenario").Update(endpoints)
-	if err != nil {
-		panic(err.Error())
-	}
-	err = re.endpointsInformer.Informer().GetIndexer().Update(updatedEndpoints)
+	now := re.env.CurrentMovementTime().UnixNano()
+	err := re.env.Plugin().Event(now, proto.EventType_CREATE, &skplug.Pod{
```

Comment:
Is this related to removing Knative? Looks like some new code.

Reply:
I just moved this code.